### PR TITLE
Add ppc64le support on multiarch docker images

### PR DESCRIPTION
**Pull request by krishvoor**
Signed-off-by: Krishna Harsha Voora <krishvoor@in.ibm.com>

## Which problem is this PR solving?
Re-adding the support for ppc64le arch, as it was missing in the following files.

## Short description of the changes
* updates `docker/Makefile`, `scripts/build-all-in-one-image.sh`, `scripts/build-upload-docker-images.sh`, `scripts/hotrod-intergration-test.sh` & `docker/debug/Dockerfile`

### CODE REVIEW

**Review by pavolloffay — DISMISSED** [review]

**Comment by jpkrohling** [comment]
Jobs restarted, looks like there was a problem logging in to Docker Hub.

File: docker/debug/Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ ARG TARGETARCH
 ENV GOPATH /go
 RUN apk add --update --no-cache ca-certificates make git
 #Once go-delve adds support for s390x (see PR #2948), remove this entire conditional.
-RUN if [[ "$TARGETARCH" != "s390x" ]] ; then \
+RUN if [[ "$TARGETARCH" != "s390x" ] || [ "$TARGETARCH" != "ppc64le" ]] ; then \
```
Comment (jpkrohling) [inline docker/debug/Dockerfile:8]: Is there a similar PR for delve to address ppc64le? If so, we need to list it here, as we are doing with s390x

Comment (krishvoor) [reply docker/debug/Dockerfile:8]: Yes, currently the same is discussed via https://github.com/go-delve/delve/issues/1564 
So do you want this to go inside the PR, just like the way it is done for s390x?

Comment (jpkrohling) [reply docker/debug/Dockerfile:8]: yes

Comment (krishvoor) [reply docker/debug/Dockerfile]: Ok, added the required details into the PR.

**Review by jpkrohling — APPROVED** [review]

**Comment by krishvoor** [comment]
Ok, looks like there was a problem logging in to Docker Hub, hence the failure and not related to PR
Can you retrigger the job, please?

**Comment by yurishkuro** [comment]
Could you please re-create the PR from a named branch in your fork, not from master? This is what's causing docker login failures.

**Comment by krishvoor** [comment]
Closing this PR.
New PR https://github.com/jaegertracing/jaeger/pull/3160

**Pull request closed without merging**